package.json

```diff
@@ -9,7 +9,10 @@
   "license": "MIT",
   "scripts": {
     "rebuild": "lerna run --stream rebuild",
-    "coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
```

Comment:
`nyc report --reporter=text-lcov > coverage.lcov` is redundant with `nyc --reporter lcovonly mocha` (`npm test`).